Doppler.UsersApi/Controllers/FeatureController.cs

```diff
@@ -1,37 +1,44 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
 using Doppler.UsersApi.DopplerSecurity;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using Doppler.UsersApi.Model;
+using AutoMapper;
 using Microsoft.Extensions.Logging;
+using Doppler.UsersApi.Infrastructure;
 
 namespace Doppler.UsersApi.Controllers
 {
     [Authorize]
     [ApiController]
     public class FeatureController
     {
-        [AllowAnonymous]
-        [HttpGet("/features")]
-        public string GetFeatures()
-        {
-            return "This method will return a list of features";
-        }
+        private readonly ILogger _logger;
+        private readonly IMapper _mapper;
+        private readonly IFeaturesRepository _featuresRepository;
 
-        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
-        [HttpGet("accounts/{accountId:int:min(0)}/features")]
-        public string GetFeaturesForAccountById(int accountId)
+        public FeatureController(ILogger<FeatureController> logger, IMapper mapper, IFeaturesRepository featuresRepository)
         {
-            return $"Will return the list of features associated to the Account ID '{accountId}'";
+            _logger = logger;
+            _mapper = mapper;
+            _featuresRepository = featuresRepository;
         }
 
         [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
         [HttpGet("/accounts/{accountname}/features")]
-        public string GetFeaturesForAccountByEmail(string accountEmail)
+        public async Task<IActionResult> GetFeaturesForAccountByEmail(string accountEmail)
         {
-            return $"Will return the list of features associated to the Account email '{accountEmail}'";
+            //TODO: validate accountEmail parameter?
```

Comment:
Very, very, very minor, but since it would be an example for future projects...

In general, we use a space between the `//` and the text.
```suggestion
            // TODO: validate accountEmail parameter?
```